sleap/gui/commands.py

```diff
@@ -3018,8 +3026,29 @@ def set_visible_nodes(
                 # We don't want to copy a PredictedPoint or score attribute.
                 x_old = copy_instance[node].x
                 y_old = copy_instance[node].y
-                x_new = x_old * scale_width
-                y_new = y_old * scale_height
+
+                # Handles if the new instance is created with right click
+                if (
+                    location is not None
+                    and offset_x is not None
+                    and offset_y is not None
+                ):
+                    x_old += offset_x
+                    y_old += offset_y
```

Comment:
If you apply the offset here, then you will be scaling the offset below resulting in the instance being offset a scaled offset amount which is undesirable.